integration/config/configtx.yaml

```diff
@@ -327,19 +327,21 @@ Orderer: &OrdererDefaults
         # implementation, we expect every replica to also be an OSN. Therefore,
         # a subset of the host:port items enumerated in this list should be
         # replicated under the Orderer.Addresses key above.
+        #
+        # NOTE: we define raft-raft port on a different port (rather than standard 7050) so we can exclusively define TLS (mandatory for raft node to raft node communication) on that port but not on the order-client port 7050 (and hence do not need client certificates)
```

Comment:
```suggestion
        # NOTE: we define raft-raft port on a different port (rather than standard 7050) so we can exclusively define TLS (mandatory for raft node to raft node communication) on that port but not on the order-client port 7050 (and hence do not need client certificates).  See also matching change in 'orderer.yaml'.
```